site/_apps/react.txt

```diff
@@ -1,10 +1,21 @@
 ---
 weight: 1
-title:  React HN
-logo: /assets/images/react-logo.svg
-lighthouse: 91/100
-interactive-em: 6.2s
-interactive-faster-3g: 4.0s
+title:  'React HN'
+library: 'React'
+library-logo: /assets/images/react-logo.svg
+module-bundling: 'Webpack'
+service-worker: 'Application Shell + data caching with Workbox'
+perfomance-patterns: 'PRPL, route-based chunking'
+server-side-rendering: 'Yes'
+api: 'Hacker News Firebase API + Node-hnapi (unoffical)'
+hosting: 'Google App Engine'
+other-details: 'Asynchronously loaded routes'
+authors: 
+    - name: 'Jonny'
+    - name: 'Addy'
+lighthouse: '91/100'
+interactive-em: '6.2s'
+interactive-faster-3g: '4.0s'
 lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170511_QV_f71a12eb44f1f058b0741db2eac3d85a&run=3
```

Comment:
@kristoferbaxter is it worth us also including an entry for your new React Hacker News implementation in here too?